.ci/pr-build-website-preview

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# Copyright 2018-2019 Datawire. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+set -o errexit
+set -o nounset
+set -o xtrace
+
+website_branch=${1:-master}
+
+rm -rf "/tmp/getambassador.io-$website_branch"
+git clone --single-branch --branch="$website_branch" https://d6e-automaton:${GH_TOKEN}@github.com/datawire/getambassador.io.git "/tmp/getambassador.io-$website_branch"
```

Comment:
This is where we clone getambassador.io.git.
Then, copy in /docs/ from ambassador.git.